### DISCUSSION TITLE
[@types/node] Align with `AbortSignal.reason`

### DISCUSSION THREAD

**Discussion by tonivj5** [DefinitelyTyped/DefinitelyTyped#64867] [Issues with a @types package] [answered]
This issue https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/64570 was parcially completed. As I mentioned into the PR, it's needed to add `abortSignal.reason` property too (https://github.com/DefinitelyTyped/DefinitelyTyped/pull/64573#issuecomment-1454219909)

Refs:
[nodejs.org/api/globals.html#abortsignalreason](https://nodejs.org/api/globals.html#abortsignalreason)
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/605a295d6198d5efe46a5f2dc76e2c16a4cc8070/types/node/globals.d.ts#L60-L65

[a comment by an automated account is not shown]

**Comment by ghost** [comment, marked as answer]
Added in #68227
